app/Http/Controllers/api/Auth/SocialLoginController.php

```diff
@@ -66,13 +62,8 @@ public function needsToCreateSocial(User $user, $service)
 
     public function getExistingUser($serviceUser, $email, $service)
     {
-        if ($service == 'google') {
```

Comment:
Please see my comment on #8

Reply:
@fritsvt I see...just replied to your comment, let me know your thoughts